vm.go

```diff
@@ -306,12 +306,32 @@ type vm struct {
 	interrupted   uint32
 	interruptVal  interface{}
 	interruptLock sync.Mutex
+
+	lastFunctionTicks uint64
 }
 
 type instruction interface {
 	exec(*vm)
 }
 
+func (vm *vm) setPrg(prg *Program) {
+	if vm.r.functionTickTrackingEnabled {
+		currentTicks := vm.r.Ticks()
+		if vm.prg != nil {
```

Comment:
What I mean by the situation of returning 0 is when vm.prg != nil. In an extreme case, in the very beginning, `vm.r.Ticks()` should return 0 because ticks is not ticking yet. `vm.lastFunctionTicks` is also 0. So line 328 will plus 0 at that time.

Reply:
Yeah I think that edge case is ok.

Reply:
I say that because I think I prefer the noop (adding 0) over the logic it would take to avoid that scenario.

Let me know if you disagree though!